app/queries/admin/enterprise_roles_query.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Admin
+  class EnterpriseRolesQuery
+    EnterpriseRoleStruct = Struct.new(:id, :user_id, :enterprise_id, :user_, :enterprise_name)
+    UserStruct = Struct.new(:id, :email)
+    EnterpriseStruct = Struct.new(:id, :name)
+
+    class << self
+      def query
+        enterprise_roles = query_enterprice_roles
+        users = query_users
+        enterprises = query_enterprises
+
+        [enterprise_roles, users, enterprises]
+      end
+
+      private
+
+      def query_enterprice_roles
```

Comment:
Small typo , it should be `query_enterprise_roles`

Reply:
And it didn't fail any specs? We need a new test covering this code.

Reply:
Ah, no, I see, the typo is in the usage as well. Just a typo, not broken code.

Reply:
:+1: nice catch, thank you